### DISCUSSION TITLE
Automatically connect new authors to users if details match

### DISCUSSION THREAD

**Discussion by andergmartins** [publishpress/PublishPress-Authors#1248] [Feature Requests]
If the selected author has the same email address of an user, map it to him. Otherwise, create a new user.

**Comment by stevejburge** [comment]
@andergmartins Two similar requests here: https://github.com/publishpress/PublishPress-Authors/issues/19

**Comment by andergmartins** [comment]
Thanks @stevejburge but the #19 is related to bulk edit posts for setting authors, and here for bulk edit the authors. We already have an option to convert the selected authors from mapped to users to guest authors, but we still don't have the inverse option.
